numpy/core/src/private/npy_config.h

```diff
@@ -99,8 +97,11 @@
 #undef HAVE_CACOSH
 #undef HAVE_CACOSHF
 #undef HAVE_CACOSHL
-#endif
-#undef TRIG_OK
+
+#endif /* __GLIBC_PREREQ(2, 16) */
+#endif /* defined(__GLIBC_PREREQ) */
+
+#endif /* defined(HAVE_FEATURES_H) */
 
 #endif /* defined(__linux__) && defined(__GNUC__) */
```

Comment:
Because we really don't care about linux and the compiler, just glibc. I checked for linux originally because I didn't know what would happen with mingw.